src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -227,6 +230,14 @@ function MoneyRequestParticipantsSelector({
 
             // We don't want to include any P2P options like personal details or reports that are not workspace chats for certain features.
             !isDistanceRequest,
+            false,
```

Comment:
We should refractor `getFilteredOptions` in future to avoid sending this much params

Reply:
Agreed. Does this JS have named params like PHP? Where we can jsut pass the params to a function by variable name?